src/services/api/notification-recipients/notification.recipients.service.ts

```
@@ -143,111 +216,78 @@ export class NotificationRecipientsService {
+      case NotificationEvent.SPACE_ADMIN_COMMUNITY_NEW_MEMBER:
+        return notificationSettings.space.admin.communityNewMember;
+      case NotificationEvent.SPACE_ADMIN_COLLABORATION_CALLOUT_CONTRIBUTION:
+        return notificationSettings.space.admin
+          .collaborationCalloutContributionCreated;
+      case NotificationEvent.SPACE_COLLABORATION_CALLOUT_CONTRIBUTION:
+        return notificationSettings.space
+          .collaborationCalloutContributionCreated;
+      case NotificationEvent.SPACE_COLLABORATION_CALLOUT_POST_CONTRIBUTION_COMMENT:
+        return notificationSettings.space
+          .collaborationCalloutPostContributionComment;
+      case NotificationEvent.SPACE_COLLABORATION_CALLOUT_CONTRIBUTION:
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)